geoproxy/main.go

```diff
@@ -115,7 +157,7 @@ func setupMap() {
 func modifyMap(conditional int) {
 	if conditional == 0 {
 		data[cr1][cr3] = "B_CONDITION_URL"
-	} else if (conditional == 1) {
+	} else if conditional == 1 {
```

Comment:
TODO (in cleanup): maintain array of condition urls, and use the indices as the conditional